test/performance/std/bit_benchmark.cpp

```diff
@@ -48,6 +48,7 @@ static void is_power_of_two_arithmetic(benchmark::State & state) {
 }
 BENCHMARK(is_power_of_two_arithmetic);
 
+#ifdef SEQAN3_DEPRECATED_310
```

Comment:
If I see this correctly, all benchmark that are seqan3 related are being removed when SEQAN3_DEPRECATED_310 is not defined. In that case we are only checking std function. May the complete file should be surrounded by a guard, so we remove it completely when we drop deprecated functionalities?

Reply:
There are still tests for intrinsics in this benchmark (`__builtin_popcountll`).